itthinx/woocommerce-product-generator
# Generate Product SKU & Support for WooCommerce CRUD

### Changes proposed in this Pull Request:

Product creation way is changed as per WooCommerce 3+ CRUD methods. Added support for woocommerce-product-sku-generator plugin. If the plugin is Active & setting is set to autogenerate SKU, it will be generated for all the Products generated via this Generator plugin.

### How to test the changes in this Pull Request:

1. Go to WooCommerce --> Product Generator tab
2. Under Single Run --> Try creating no. of Products
3. Under Continuous AJAX Run --> Try creating products via AJAX

This should create Simple Products as in the earlier versions.

### Other information:

- [x] Tested changes on the local system.

### Changelog entry
WooCommerce CRUD Product creation support.
Support for SKU Generator for woocommerce-product-sku-generator plugin.
